scripts/calculate_metrics.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import json
+import os
+
+import yaml
+from functions import opts2dmpairArgs
+
+from modsim2.data.loader import DMPair
+
+
```

Comment:
we use logging level of INFO for some messages in the called code. We should set logging level here (either fixed choice or by user option)

Reply:
I've fixed the logging level to DEBUG